docker_test.go

```diff
@@ -54,9 +55,11 @@ func runDockerPushTest(imageName, module string, withModule bool, t *testing.T)
 	// Push docker image using docker client
 	if withModule {
 		artifactoryCli.Exec("docker-push", imageTag, *tests.DockerTargetRepo, "--build-name="+tests.DockerBuildName, "--build-number="+buildNumber, "--module="+module)
+		inttestutils.ValidateGeneratedBuildInfo(t, tests.DockerBuildName, buildNumber, []string{module}, buildinfo.Docker)
```

Comment:
Let's add the above validation also to the ```else``` section of this ```if```. We'll need to use the default module name value there.